src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -39,8 +42,33 @@
      * Андрей - 600
      * София - 900</p>
      */
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo)
             throws Exception {
-        return null;
+        DateTimeFormatter d = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
make formatter constant and rename it, please, due to this article: [https://mate-academy.github.io/style-guides/java/java.html](url)